### ISSUE TITLE
Deploying new version results in the errors due to improper clusterrole

### ISSUE THREAD

**Issue by mandoddi** [reactiveops/polaris#153]
so i deployed a new version, 0.2 of polaris and it resulted in the below error.. i was able to solve by changing the clusterrole permissions for polaris-dashboard. below is the error for the reference.

`time="2019-06-21T21:12:50Z" level=info msg="Starting Polaris dashboard server on port 8080"
time="2019-06-21T21:13:19Z" level=error msg="Error fetching Nodes nodes is forbidden: User \"system:serviceaccount:polaris:polaris-dashboard\" cannot list resource \"nodes\" in API group \"\" at the cluster scope"
time="2019-06-21T21:13:19Z" level=error msg="Error fetching Kubernetes resources nodes is forbidden: User \"system:serviceaccount:polaris:polaris-dashboard\" cannot list resource \"nodes\" in API group \"\" at the cluster scope"`

**Comment by bobby-brennan** [comment]
Thanks for the report! I was able to reproduce an issue with the config in our release config by running:

```
kubectl apply -f https://github.com/reactiveops/polaris/releases/latest/download/dashboard.yaml
```

That file has been fixed manually for now (as well as webhook.yaml). However, I was seeing an issue related to `statefulsets` rather than `nodes` as you show above. Are you still seeing the issue?

This exposes an issue with our release process. The Polaris chart was fixed on master at github.com/reactiveops/charts, but CI/CD uses the `polaris-latest` branch, which didn't get the fix.  I've opened https://github.com/reactiveops/polaris/issues/154 to track.
